cpu/nrf52/radio/nrf802154/nrf802154.c

```diff
@@ -131,6 +131,26 @@ static void _enable_tx(void)
     NRF_RADIO->TASKS_START = 1;
  }
 
+/**
+ * @brief basic ieee802.15.4 data frame validation
+ */
+static bool _data_frame_filter(void)
+{
+    size_t psdu_len = rxbuf[0];
```

Comment:
does this rly need an own variable?

Reply:
> does this rly need an own variable?

No. I've made it a separate variable to improve readability. The compiler is smart enough to not let it impact the code size.